includes/Modules/Analytics_4/Custom_Dimensions_Data_Available.php

```diff
@@ -111,9 +111,14 @@ public function set_data_available( $custom_dimension ) {
 	 * Resets the data available state for all custom dimensions.
 	 *
 	 * @since 1.113.0
+	 * @since n.e.x.t Added optional $custom_dimensions parameter.
+	 *
+	 * @param array $custom_dimensions Optional. List of custom dimension slugs to reset.
 	 */
-	public function reset_data_available() {
-		foreach ( self::CUSTOM_DIMENSION_SLUGS as $custom_dimension ) {
+	public function reset_data_available(
```

Comment:
Please add/update the test coverage for this method accordingly, thanks!

Reply:
Added, thank you!